### PR TITLE
gRPC CLI type command

### PR DESCRIPTION
**Pull request by y-zeng**
Add type command for gRPC CLI.

```
grpc_cli type <address> <type>
    <address>                ; host:port
    <type>                   ; Protocol buffer type name
```

### CODE REVIEW

**Comment by yang-g** [comment]
test this please

File: test/cpp/util/grpc_tool.cc [hunk 1]
```diff
@@ -257,6 +257,33 @@ bool GrpcTool::Help(int argc, const char** argv, const CliCredentials cred,
   return true;
 }
 
+bool GrpcTool::PrintType(int argc, const char** argv, CliCredentials cred,
```
Comment (yang-g) [inline test/cpp/util/grpc_tool.cc]: const CliCredentials& cred ?

File: test/cpp/util/grpc_tool_test.cc [hunk 2]
```diff
@@ -109,13 +109,21 @@ class GrpcToolTest : public ::testing::Test {
 
   void ShutdownServer() { server_->Shutdown(); }
 
+  void ExitWhenError(int argc, const char** argv, const CliCredentials cred,
```
Comment (yang-g) [inline test/cpp/util/grpc_tool_test.cc]: const CliCredentials& ?

File: test/cpp/util/grpc_tool.cc (hunk 1 above)
Comment (y-zeng) [inline test/cpp/util/grpc_tool.cc]: Fixed. Thanks for catching this!

File: test/cpp/util/grpc_tool_test.cc (hunk 2 above)
Comment (y-zeng) [inline test/cpp/util/grpc_tool_test.cc]: Fixed.

**Comment by y-zeng** [comment]
Test failures seem irrelevant. Only `grpc_tool_test` is affected.

**Pull request merged**